install.sh

```diff
@@ -81,26 +84,57 @@ while [ "$#" -gt 0 ]; do
 	case $1 in
 		-h) printf "\nUsage: $0\n"
 			printf "\n    [-c] catkin_workspace (Recommend: ~/mil_ws)\n"
+			printf "\n    [-a] Installs everything needed for all MIL projects\n"
+			printf "\n    [-s] Installs everything needed for SubjuGator 8\n"
+			printf "\n    [-n] Installs everything needed for Navigator\n"
 			printf "\n    example: ./install.sh -c ~/mil_ws\n"
 			printf "\n"
 			exit 0
 			;;
 		-c) CATKIN_DIR="$2"
 			shift 2
 			;;
+		-a) INSTALL_ALL=true
+			shift 1
+			;;
+		-s) INSTALL_SUB=true
+			shift 1
+			;;
+		-n) INSTALL_NAV=true
+			shift 1
+			;;
 		-?) instwarn "Option $1 is not implemented"
 			exit 1
 			;;
 	esac
 done
 
+if !($INSTALL_ALL || $INSTALL_SUB || $INSTALL_NAV); then
+	echo "A MIL project must be selected for install"
+	echo "Run ./install.sh -h for more information"
+	exit 1
+elif ($INSTALL_ALL); then
+	INSTALL_SUB=true
```

Comment:
Wasn't sure if people wanted this as a default, but it would be easy to set that up